app/controllers/support/allocations_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Support
+  class AllocationsController < ApplicationController
+    def index
+      @providers_with_allocations = filtered_providers_with_allocations.page(params[:page] || 1)
+    end
+
+    def show
+      allocation
+    end
+
+  private
+
+    def allocation
+      @allocation ||= Allocation.find(params[:id])
+    end
+
+    def filtered_providers_with_allocations
+      Support::Providers::Filter.call(providers: find_providers_with_allocations, filters: filters)
+    end
+
+    def find_providers_with_allocations
+      Provider.where(id: Allocation.current_allocations.select(:provider_id)).order(:provider_name)
```

Comment:
Could this be a scope on Provider, eg `Provider.with_allocations_for_current_cycle_year`?

The implementation could also be a join between providers and allocations, along with a where the scopes to the current cycle year.